src/lib/constants.js

```diff
@@ -68,6 +68,9 @@ export const TRACE_TO_AXIS = {
     'histogram',
     'histogram2d',
     'histogram2dcontour',
+    'carpet',
+    'scattercarpet',
+    'contourcarpet',
```

Comment:
but so, should we also add parcoords and other unsupported types, just in case someone creates them from the API, 

or I guess that just returning null in getAllAxes takes care of the panel not erroring out for those traces?

Reply:
returning null causes the Axes panel to be hidden :)